setup.py

```diff
@@ -29,7 +29,7 @@
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
     'swagger_spec_validator>=2.1.0',
-    'pytest-reana>=0.5.0.dev20190321',
+    'pytest-reana==0.5.0.dev20190404',
```

Comment:
Can you set it back to `>=`? I have deleted the `0.5.0.dev201903211` release which was making it install the wrong version. 🎊